src/Apps/IdentityVerification/IdentityVerificationApp/CompleteFailed.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Button, Link, Sans, Serif } from "@artsy/palette"
+import React from "react"
+
+export const CompleteFailed: React.FC = () => {
+  return (
+    <Box px={[2, 3]} mb={6} mt={4}>
+      <Box
```

Comment:
I'm not sure if you have to nest `<Box>`es here. You could find [a refactoring example here](https://github.com/artsy/reaction/pull/3282/files#diff-b6e1258870cda8e1ddc88e37a1932561L129-R123).